docs/docs/data-structures/tree/index.mdx

```diff
@@ -20,467 +18,45 @@ A `SharedTree` has the following characteristics:
-
-###### Insert methods
-
-Array nodes have three methods that insert new items into the node. Note that in all of the following, the `T` can be any type that conforms to the array node's schema. For example, if the schema was defined with `class MyArray extends sf.array([sf.number, sf.string]);`, then `T` could be `number` or `string`.
-
-```typescript
-insertAt(index: number, value: Iterable<T>)
-```
-
-Inserts the provided value(s) at the specified `index`. If the `index` is greater than the length of the array, the items are inserted at the end of the array.
-
-```typescript
-insertAtStart(value: Iterable<T>)
-```
-
-Inserts the provided value(s) at the start of the array. This is sugar for `insertAt(0, …)`.
-
-```typescript
-insertAtEnd(value: Iterable<T>)
-```
-
-Inserts the provided value(s) at the end of the array. This is syntactic sugar for `insertAt(Infinity, …)`.
-
-###### Remove methods
-
-Array nodes have two methods that remove items from the node. Note the following about these methods:
-
--   Removed items are saved internally for a time in case they need to be restored as a result of an undo operation.
--   A removed item may be restored as a result of a simultaneous move operation from another client. For example, if one client removes items 3-5, and another client simultaneously moves items 4 and 5, then, if the move operation is ordered last, only item 3 is removed (items 4 and 5 are restored and moved to their destination by the move operation). If the remove operation is ordered last, then all three items will be removed, no matter where they reside.
--   Removal of items never overrides inserting (or moving in) items. For example, if one client removes items 10-15, and another client simultaneously inserts an item at index 12, the original items 10-15 are removed, but new item is inserted between item 9 and the item that used to be at index 16. This is true regardless of the order of the remove and insert operations.
-
-For the meaning of "simultaneously", see [Types of distributed data structures](../overview.mdx).
-
-```typescript
-removeAt(index: number)
-```
-
-Removes the item at the given `index`.
-
-```typescript
-removeRange(start?: number, end?: number)
-```
-
-Removes the items indicated by the `start` index (inclusive) and `end` index (exclusive). If the end index is omitted, every item from the start index to the end of the array is removed. If the start index is omitted, it defaults to 0. So, calling `removeRange()` removes all the items in the array.
-
-###### Move methods
+To get started working with `SharedTree` in your application, read this [quick start guide](../../start/tree-start.mdx).
+The major programming tasks for using `SharedTree`s that are described in this doc are:
```

Comment:
I really like the approach of organizing information in terms of activities that the user might engage in, but this page doesn't actually cover these programming tasks anymore. It's just linking to pages that do, so that seems confusing/misleading and is just taking up space since the list of links below already comes with descriptions.